dataeng/jobs/analytics/LoadPaypalCaseReportToVertica.groovy

```diff
@@ -0,0 +1,27 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+
+class LoadPaypalCaseReportToVertica{
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("Load-Paypal-CaseReport-to-Vertica"){
```

Comment:
I think we normally use lowercase for job titles, right?

Reply:
right. Will fix this.